test_elasticsearch/test_server/test_clients.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from . import ElasticsearchTestCase
```

Comment:
I renamed the file from `test_client.py` to `test_clients.py` because I had trouble with this import when running the test from the IDE:

```
Traceback (most recent call last):
  File "/Users/daniel/Library/Application Support/IdeaIC2019.3/python-ce/helpers/pycharm/_jb_unittest_runner.py", line 35, in <module>
    main(argv=args, module=None, testRunner=unittestpy.TeamcityTestRunner, buffer=not JB_DISABLE_BUFFERING)
  File "/usr/local/var/pyenv/versions/3.8.0/lib/python3.8/unittest/main.py", line 100, in __init__
    self.parseArgs(argv)
  File "/usr/local/var/pyenv/versions/3.8.0/lib/python3.8/unittest/main.py", line 147, in parseArgs
    self.createTests()
  File "/usr/local/var/pyenv/versions/3.8.0/lib/python3.8/unittest/main.py", line 158, in createTests
    self.test = self.testLoader.loadTestsFromNames(self.testNames,
  File "/usr/local/var/pyenv/versions/3.8.0/lib/python3.8/unittest/loader.py", line 220, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/usr/local/var/pyenv/versions/3.8.0/lib/python3.8/unittest/loader.py", line 220, in <listcomp>
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/usr/local/var/pyenv/versions/3.8.0/lib/python3.8/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/Users/daniel/Projects/elasticsearch-py/test_elasticsearch/test_client/__init__.py", line 5, in <module>
    from ..test_cases import TestCase, ElasticsearchTestCase
ValueError: attempted relative import beyond top-level package
```

This seems to be caused by the fact that there is also a package named `test_client` and it does not work well with relative imports. Another option would have been to change this to an absolute import but this does not seem to be match the project's style so I avoided this.